db/changes/116_add_car_table.sql

```diff
@@ -0,0 +1,48 @@
+-- Create new tables for Car and Cars milage
+CREATE TABLE IF NOT EXISTS companycar (
+CompanyCarID int(11) NOT NULL AUTO_INCREMENT,
+CarName varchar(50) NOT NULL DEFAULT '',
+TS timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+Active tinyint(4) NOT NULL DEFAULT '0',
+ValidFrom datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+ValidTo datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+Description text,
+CarCode varchar(10) DEFAULT NULL,
+PurchasePrice decimal(16, 2) DEFAULT '0',
+PurchaseAddCost int(11) DEFAULT '0',
+PurchaseYear smallint(6) NOT NULL DEFAULT '0',
+CurrentPrice int(11) NOT NULL DEFAULT '0',
```

Comment:
Remove
+PurchaseAddCost int(11) DEFAULT '0',
+PurchaseYear smallint(6) NOT NULL DEFAULT '0',
+CurrentPrice int(11) NOT NULL DEFAULT '0',